esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -118,7 +121,9 @@ define(
     Copied: "Copiado",
     Copy: "Copia",
     CopyToClipboard: "Copiar al portapapeles",
+    CopyURLToClipboard: "Copiar URL al portapapeles",
```

Comment:
Sorry, I meant to put "Copia" on these.

Reply:
should it be fixed on the two below it as well CopyWUIDs and CopyWUIDsToClipboard as well?

Reply:
Should be fixed on both additions.